src/main/kotlin/com/example/gymi/domain/notice/service/impl/EditNoticeServiceImpl.kt

```diff
@@ -0,0 +1,66 @@
+package com.example.gymi.domain.notice.service.impl
+
+import com.example.gymi.domain.file.service.FileService
+import com.example.gymi.domain.file.util.FileUtil
+import com.example.gymi.domain.notice.entity.Notice
+import com.example.gymi.domain.notice.entity.NoticeFile
+import com.example.gymi.domain.notice.exception.NoticeNotFoundException
+import com.example.gymi.domain.notice.presentation.data.dto.EditNoticeDto
+import com.example.gymi.domain.notice.presentation.data.request.EditNoticeRequestDto
+import com.example.gymi.domain.notice.repository.NoticeFileRepository
+import com.example.gymi.domain.notice.repository.NoticeRepository
+import com.example.gymi.domain.notice.service.EditNoticeService
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import org.springframework.web.multipart.MultipartFile
+
+@Service
+@Transactional(rollbackFor = [Exception::class])
+class EditNoticeServiceImpl(
+    private val noticeRepository: NoticeRepository,
+    private val noticeFileRepository: NoticeFileRepository,
+    private val fileService: FileService,
+    private val fileUtil: FileUtil
+) : EditNoticeService {
+
+    override fun execute(id: Long, editNoticeRequestDto: EditNoticeRequestDto, multipartFile: List<MultipartFile>?): Notice {
+
+        val noticeInfo: Notice = noticeRepository.findByIdOrNull(id)
+            ?: throw NoticeNotFoundException()
+
+        val noticeFileInfo: List<NoticeFile> = noticeFileRepository.findAllByNoticeId(noticeInfo.id)
+
+        toDto(editNoticeRequestDto)
+            .let { noticeInfo.editNotice(title = it.title, content = it.content) }
```

Comment:
저기서 let을 사용하기보단 
```suggestion
        var editNoticeDto = toDto(editNoticeRequestDto)
        noticeInfo.editNotice(title = editNoticeDto.title, content = editNoticeDto.content)
```
이렇게 하는게 더 직관적인거 같아요

Reply:
수정했습니다!